ulid_test.go

```diff
@@ -390,6 +390,32 @@ func TestOverflowHandling(t *testing.T) {
 	}
 }
 
+func TestScanBytes(t *testing.T) {
```

Comment:
These two tests are mostly identical. Could we merge them into a single table driven test? Something like:

```go
id := ulid.MustNew(123, crand.Reader)

for _, tc := range []struct{
    name string
    in interface{}
    out ULID
    err error
} {
    {"string", id.String(), id, nil},
    {"bytes", id[:], id, nil},
    {"nil", nil, ULID(nil), nil},
    {"other", 44, nil, ErrScanValue},
} {
    tc := tc
    t.Run(tc.name, func(t *testing.T) {
        t.Parallel()

        var out ulid.ULID
        err := out.Scan(tc.in)
        if got, want := out, tc.out; !got.Equal(want) {
            t.Errorf("got ULID %s, want %s", got, want)
        }

         if got, want := fmt.Sprint(err), fmt.Sprint(tc.err); got != want {
             t.Errorf("got err %q, want %q", got, want)
         }
    })
    
}
```